css/main.css

```diff
@@ -0,0 +1,1109 @@
+html {
+    padding: 0;
+    margin: 0;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+body {
+    display: flex;
+    min-height: 100px;
+    background-color: #000;
+    margin: 0;
+    padding: 0;
+}
+
+.logo {
+    height: 20px;
+    width: 20px;
+}
+
+.logo img {
+    position: absolute;
+    height: 70px;
+    width: 70px;
+    left: 40px;
+    top: 18px;
+}
+
+.nav {
+    display: flex;
+    align-items: flex-end;
+    position: absolute;
+    top: 48px;
+}
+
+.nav_right-20px {
```

Comment:
You mixed up delimiters. Please, read else one time [this](https://ru.bem.info/methodology/html/). Fix it on all over the layout. And name of the class keep `20px` value, but actually it has `300px`.